package.serve.json

```diff
@@ -9,17 +9,74 @@
   },
   "scripts": {
     "setup-serve": "./bin/download_settings_controls.js && ./bin/setup_autodeploy.js",
-    "serve": "phonegap --verbose serve"
+    "serve": "webpack && concurrently -k \"phonegap --verbose serve\" \"webpack --watch\"",
+    "serve-only": "phonegap --verbose serve"
   },
   "devDependencies": {
-    "phonegap": "9.0.0+cordova.9.0.0",
-    "cordova": "^11.1.0",
+    "@babel/core": "^7.21.3",
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
+    "@babel/plugin-proposal-object-rest-spread": "^7.20.7",
+    "@babel/preset-env": "^7.21.4",
+    "@babel/preset-flow": "^7.21.4",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/preset-typescript": "^7.21.4",
     "@ionic/cli": "6.20.8",
-    "bower": "1.8.14"
+    "babel-loader": "^9.1.2",
+    "babel-plugin-optional-require": "^0.3.1",
+    "concurrently": "^8.0.1",
+    "cordova": "^11.1.0",
+    "css-loader": "^6.7.4",
+    "exports-loader": "^4.0.0",
+    "expose-loader": "^4.1.0",
+    "file-loader": "^6.2.0",
+    "phonegap": "9.0.0+cordova.9.0.0",
+    "process": "^0.11.10",
+    "style-loader": "^3.3.3",
+    "ts-loader": "^9.4.2",
+    "typescript": "^5.0.3",
+    "url-loader": "^4.1.1",
+    "webpack": "^5.0.1",
+    "webpack-cli": "^5.0.1"
   },
   "dependencies": {
+    "angular": "1.6.7",
+    "angular-animate": "1.6.7",
+    "angular-local-storage": "^0.7.1",
+    "angular-nvd3": "^1.0.7",
+    "angular-qrcode": "https://github.com/sebastianbarry/angular-qrcode.git#v7.3.0",
+    "angular-sanitize": "1.6.7",
+    "angular-simple-logger": "^0.1.7",
+    "angular-translate": "^2.18.1",
+    "angular-translate-loader-static-files": "^2.18.1",
+    "angular-ui-router": "0.2.13",
+    "animate.css": "^3.5.2",
+    "bottleneck": "^2.19.5",
+    "core-js": "^2.5.7",
     "fs-extra": "^9.0.1",
+    "i18next": "^22.5.0",
+    "install": "^0.13.0",
+    "ionic-datepicker": "1.2.1",
+    "ionic-toast": "^0.4.1",
+    "jquery": "^3.1.0",
+    "jquery-xml2json": "^0.0.8",
     "klaw-sync": "^6.0.0",
+    "leaflet": "^0.7.7",
+    "leaflet-plugins": "^3.0.0",
+    "leaflet.awesome-markers": "^2.0.5",
+    "messageformat": "^2.3.0",
+    "moment": "^2.29.4",
+    "moment-timezone": "^0.5.43",
+    "ng-i18next": "^1.0.7",
+    "npm": "^9.6.3",
+    "nvd3": "^1.8.6",
```

Comment:
I'm not sure. I was hoping you could tell me why we're using `nvd3` and not simply `d3.js`.
I'm not sure `nvd3` is even maintained anymore - in the few places we show charts maybe it's worthwhile to just switch to the more-popular and better-supported `d3.js`

Reply:
I don't remember any more. I would have to look at the blame from when I added it to see if there were any clues.
OK so it looks like this was part of https://github.com/e-mission/e-mission-phone/pull/69, where I ported over the stats screen from the server.

On the server, we added it in 
https://github.com/e-mission/e-mission-server/pull/310/commits/467272aee79e29d0f565ad73ac0ad111713276b8#diff-4ae9bb5dbf0fc9283c982c8472a8a2a7012a1fb5c3a07a6c2d4ad9a1e15177b4

which says what we do, but now why we chose the nvd3 wrapper.

I probably just picked it because it seemed easier to use, or it was recommended by somebody at the lab.
And at the time, we didn't know how it would be maintained.